tests/db/test_db.py

```diff
@@ -6,12 +9,43 @@
 class ModelTest(BaseModel):
     """Model for testing purposes"""
 
+    class Schema(BaseModel.Schema):
+        """Schema for testing purposes"""
+
+        test_value = marsh.fields.Integer(required=True)
+
+        def _make(self, data: dict) -> "ModelTest":
+            return ModelTest(**data)
+
     test_value = IntegerField()
 
+    def __init__(self, test_value, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self.test_value = test_value
+
+
+class TestDB:
+    """Test the database connection and the serialization/deserialization"""
+
+    def setup_method(self):
+        db.create_tables([ModelTest])
+
+    def teardown_method(self):
+        db.drop_tables([ModelTest])
+
+    def test_db_connection_workflow(self):
+        # I just wanted to test if the db is working
+        ModelTest.create(test_value=1).save()
+        test_obj = ModelTest.select().where(ModelTest.test_value == 1).first()
+        assert test_obj.test_value == 1
```

Comment:
```suggestion
        test_obj = ModelTest.create(test_value=1)
        test_obj.save()
        returned_obj = ModelTest.select().where(ModelTest.test_value == 1).first()
        assert test_obj == returned_obj
```
It may be better to check for the identity of the returned object and not fot the value.

Reply:
I would keep comparing each value, because it's not the same object. Querying a ModelTest instance creates a new object with the same value.